test/testdata/resolver/type_member_parent_missing.rb

```diff
@@ -11,7 +11,7 @@ class Bar1 < Foo1
 # ^^^^^^^^^^^^^^^^^ error: Missing definition for abstract method `Enumerable#each`
     Elem = type_member(:out)
   # ^^^^^^^^^^^^^^^^^^^^^^^^ error: Classes can only have invariant type members
-  # ^^^^^^^^^^^^^^^^^^^^^^^^ error: Type variance mismatch with parent `Foo1`
+  # ^^^^^^^^^^^^^^^^^^^^^^^^ error: Type variance mismatch for `Elem` with parent `Foo1`. Child `Bar1` should be `:invariant` or invariant, but it is `:out`
```

Comment:
Sorry for not noticing this earlier: "`:invariant` or invariant" is redundant and also confusing, because `:invariant` is not a valid variance annotation.

Reply:
I went ahead and pushed up a fix for this to your branch.